crates/re_space_view_spatial/src/mesh_loader.rs

```diff
@@ -233,7 +233,7 @@ fn mesh_texture_from_tensor_data(
         Ok(re_renderer::resource_managers::Texture2DCreationDesc {
             label: "mesh albedo texture from tensor data".into(),
             data,
-            format: re_renderer::wgpu::TextureFormat::Rgba8UnormSrgb,
+            format: re_renderer::external::wgpu::TextureFormat::Rgba8UnormSrgb,
```

Comment:
I suggest we re-export common types that are part of our public API

```suggestion
            format: re_renderer::TextureFormat::Rgba8UnormSrgb,
```

Reply:
in other places we tended to directly import wgpu instead 🤔. I have a bit of a bad feeling about exposing `TextureFormat` directly given that re_renderer leaks a lot of wgpu types in a lot of its APIs - I'm unsure where to stop.

Reply:
I would make the cutoff at "very commonly used" or not, but 🤷  your call